hack/builder/Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN dnf install -y dnf-plugins-core && \
         sudo \
         gcc \
         gcc-c++ \
+        glibc-static \
+        libstdc++-static.x86_64 \
```

Comment:
I don't think that we need anything from c++ here. Can it be that you can already get rid of this dependency?

Reply:
I tried to compile without it and it did not work, I think the problem that bazel does not have separate rules for C only, it has only `cc_binary`, so it compiled as C++ with C++ flags and one of the flags is `-lstd++` and it requires `libstdc++-static.x86_64`